src/models/sponsor.ts

```diff
@@ -13,7 +13,25 @@ export interface Sponsor {
 
 export enum SubscriptionTier {
 	BRONZE,
+	SILVER,
 	GOLD,
 	PLATINUM,
-	SILVER,
+	INVALID,
+}
+
+export function getTierString(tier: SubscriptionTier) {
```

Comment:
We can refactor these functions to be more simple.
`export enum SubscriptionTier {
  BRONZE = "bronze",
  SILVER = "silver",
  GOLD = "gold",
  PLATINUM = "platinum",
  INVALID = "invalid",
}

export function getTierString(tier: SubscriptionTier) {
  return tier.charAt(0).toUpperCase() + tier.slice(1);
}

export function getTierFromString(tier: string) {
  tier = tier.toLowerCase();
  return SubscriptionTier[tier];
}
`